easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-GCCcore-6.4.0.eb

```diff
@@ -18,7 +18,12 @@ toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969']
+checksums = [
+    '954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969',  # autoconf-2.69.tar.gz
+    '545b602ccfb3e8f27bc41b5f92a2e6aee672a2eaad73760f3fd31102cf2caf2b',  # Autoconf-2.69_config.guess_ppc64le.patch
+]
+
+patches = ['Autoconf-2.69_config.guess_ppc64le.patch']
```

Comment:
@omula Please use `%(version)s` rather than hardcoding `2.69` here, and move this line up above `checksums =`